server/src/main/java/io/crate/execution/dml/ObjectIndexer.java

```diff
@@ -196,7 +194,8 @@ private void addNewColumns(Map<String, Object> value,
                 nullable,
                 storageSupport.docValuesDefault(),
                 position,
-                defaultExpression
```

Comment:
The use of local variables here was intentional to make it easier to see what the parameters are